lib/ClangImporter/ImportName.cpp

```diff
@@ -1094,7 +1094,7 @@ ImportedName NameImporter::importNameImpl(const clang::NamedDecl *D,
   // Objective-C categories and extensions don't have names, despite
   // being "named" declarations.
   if (isa<clang::ObjCCategoryDecl>(D))
-    return {};
+    return ImportedName();
```

Comment:
I'm okay with this change, but is it really necessary? This seems like a pretty common C++11 feature that will keep slipping in.